### DISCUSSION TITLE
如何彻底卸载v2ray

### DISCUSSION THREAD

**Discussion by BobNeer** [v2fly/v2ray-core#1600] [Q&A]
我安装之后无法正常启动，打算卸载重装，但是按照https://github.com/v2ray/v2ray-core/issues/187 进行卸载之后，输入“systemctl status v2ray”仍然能显示信息，是否没有卸载成功呢？系统：centos 7.6 和8.5

**Comment by BobNeer** [comment]
centos 7.6是腾讯云的轻量应用，是否是由于腾讯云封禁导致服务无法启动
centos8.5是hostwinds的vps，用来搭梯子的，之前用
https://git.io/v2ray-setup.sh
一键安装的。

今天脑抽，以为v2ray客户端和服务端是两个东西，用
https://raw.githubusercontent.com/v2fly/fhs-install-v2ray/master/install-release.sh
又安装了一遍，导致出现了冲突，现在cent8.5上服务可以用了，只是v2ray命令进去显示服务是否启动的状态显示有问题

**Reply by BobNeer** [reply]
https://ruiruigeblog.com/2017/01/21/CentOS%E4%BD%BF%E7%94%A8systemctl%E5%BD%BB%E5%BA%95%E5%88%A0%E9%99%A4%E6%9C%8D%E5%8A%A1/
找到这篇博文，照着操作后安装正常了
关键是要执行 rm -rf /etc/systemd/system/v2ray* （有v2ray和v2ray@各一个.d文件夹和.service文件）

**Reply by rchangl** [reply]
安装文件sh里面有关于/etc/systemd/的，打开看看他的操作再逆向操作就没遗漏了
